docs/source/50_cli.md

```diff
@@ -330,6 +311,19 @@ options](#default-args). Additionally, `adam2vcf` takes the following options:
 * `-single`: Saves the VCF file as headerless shards, and then merges the
   sharded files into a single VCF.
 
```

Comment:
Needs a line like

* `-stringency`: Sets the validation stringency for various checks. Defaults to STRICT. See validation stringency for more details.